rust_src/src/eval_macros.rs

```diff
@@ -17,13 +17,15 @@ macro_rules! xsignal {
     ($symbol:expr) => {
         #[allow(unused_unsafe)]
         unsafe {
-            crate::remacs_sys::Fsignal($symbol, crate::remacs_sys::Qnil);
+            crate::eval::Fsignal($symbol, crate::remacs_sys::Qnil);
```

Comment:
Call `signal` not `Fsignal`.

Reply:
Done. Also I moved the panic if it's called in `cfg(test)` to the signal function.
make check still errors with "undefined reference to `signal_or_quit`", I'm trying to find out why